system-int-test/test/cmr/system_int_test/search/variable/concept_retrieval_test.clj

```diff
@@ -0,0 +1,109 @@
+(ns cmr.system-int-test.search.variable.concept-retrieval-test
+  "Integration test for variable retrieval via the /concepts/:concept-id and
+  /concepts/:concept-id/:revision-id endpoints."
+  (:require
+   [cheshire.core :as json]
```

Comment:
There are a ton of unused requires here. Can you use something like slamhound on this file to get rid of all the unused requires?

I even see cmr.common.mime-types in there twice 😄 . Looks like a copy/paste from somewhere that might also need to be fixed.